root/inc/release-tools.html

```diff
@@ -21,6 +21,11 @@
     </a>
 </li>
 <li>
+	<a href="#" onclick="show_install_instructions(); return true;">
```

Comment:
Also, this should be a <button> element, since it isn't really a link.  Applying the class btn-link should make it look like the other links.